### ISSUE TITLE
TODO: Extends tests to run integration code

### ISSUE THREAD

**Issue by KinoxKlark** [pyimgui/pyimgui#245]
As suggested by @connorjak in https://github.com/pyimgui/pyimgui/issues/243#issuecomment-956530975, we should add tests for each integration that would launch the integration and exercise it (call functions and action callbacks).

Notes:
* Tests of actual functionalities are automatically parsed from the examples in the documentation. No need to run every single example for every integration.
* The `tests/` directory contains miscellaneous tests. Integration tests could go here.
* I don't think that a parsing and automatic detection of functions to call is necessary here. There are not so many cases and we can map and maintain them manually. Feel free to suggest otherwise.

**Comment by connorjak** [comment]
Right, integration tests are much less important to completely cover than unit tests. Integration tests (in general) are mostly a canary to tell you if something outside your usual development environment isn't integrating well with the new changes you have made, before users tell you that.

**Comment by learn-more** [comment]
With the correct config, pylint is able to pick this up:
```
py -m pylint -E .\src\imgui\integrations\pyglet.py -v --extension-pkg-whitelist=imgui
************* Module imgui.integrations.pyglet
src\imgui\integrations\pyglet.py:108:34: E1101: Module 'imgui' has no 'CONFIG_NO_MOUSE_CURSOR_CHARGE' member; maybe 'CONFIG_NO_MOUSE_CURSOR_CHANGE'? (no-member)
```

(The `--extension-pkg-whitelist` option is required because this package contains an executable module, which it should load)
